extension/js/common/composer.ts

```diff
@@ -436,6 +438,17 @@ export class Composer {
     }
   }
 
+  private checkReciepientsKeys = async () => {
+    for (const recipientEl of $('.recipients span.no_pgp')) {
+      const email = $(recipientEl).text().trim();
+      const res = await this.lookupPubkeyFromDbOrKeyserverAndUpdateDbIfneeded(email);
```

Comment:
Here I recommend to check db directly, without checking attester. After all, the imported key should by now already be in the local db.

(as implemented now, If you have 3 recipients who are all gray, and add one pubkey, two of them will be re-checked on Attester unnecessarily)

Something like (please test):

```js
const [dbContact] = await this.app.storageContactGet([email]);
if(dbContact) {
  $(recipientEl).removeClass('no_pgp');
  await this.renderPubkeyResult(recipientEl, email, dbContact);
}
```